readme.txt

```diff
@@ -33,8 +38,8 @@ Usage:
         3. a verbose json report which delineates all of the issues found by the scans. This can be found in the reports directory 
     After every session of scans a new set of these three reports are generated. The latter two being files differentiated from other scans by timestamps. 
 
-
-
+WARNINGS    
+    Currently it is not recommended to run this program parallely due to certain static analyzers interacting withthem selves
 
```

Comment:
I remember we discussed about adding additional details around debugging sonarqube server status, which port sever is running command to check the status. How to bring up server manually, if needed etc...

You can separate heading about Sonarqube and these details